modules/sdk/project-templates/project-templates/src/main/java/com/liferay/project/templates/internal/util/FileUtil.java

```diff
@@ -211,6 +211,30 @@ public static Path getFile(Path dirPath, String glob) throws IOException {
 		return null;
 	}
 
+	public static Path getFile(Path dirPath, String glob, String regex)
+		throws IOException {
+
+		try (DirectoryStream<Path> directoryStream = Files.newDirectoryStream(
+				dirPath, glob)) {
+
+			Iterator<Path> iterator = directoryStream.iterator();
+
+			while (iterator.hasNext()) {
+				Path path = iterator.next();
+
+				Path fileName = path.getFileName();
```

Comment:
The variable name should end with the class name for readability (i.e. `fileNamePath`).